src-electron/util/env.js

```diff
@@ -611,6 +612,79 @@ export function versionsCheck() {
  *
  * @returns full path to HTTP static content
  */
-export function httpStaticContent() {
+function httpStaticContent() {
   return httpStaticContentPath
 }
+
+/**
+ * Format a message with emoji if enabled, without if disabled.
+ * @param {string} emoji - the emoji character
+ * @param {string} message - the text message
+ * @returns {string} formatted message
+ */
+function formatEmojiMessage(emoji, message) {
+  if (emojiUtil.isEmojiDisabled()) {
+    return message
+  }
+  return `${emoji} ${message}`
+}
+
+/**
+ * Set emoji disabled state (mainly for testing)
+ * @param {boolean} disabled - whether to disable emojis
+ */
+function setNoEmoji(disabled) {
```

Comment:
What is calling this function? Either I don't have all files open and my browser search is failing me. Either way why not call emojiUtil.setEmojiDisabled instead of having this additional function

Reply:
nothing, just didn't cleanup, good catch